### PR TITLE
Core_workflows.py: Added method to simulate network delay and packet loss

### PR DESCRIPTION
**Pull request by pdhiran**
…loss

Added new method to simulate network delays and packet loss on the ceph nodes & clients. Added the delays during rebalance scenarios for:
1. Regular RHCS cluster during PG split-merge -> recovery
2. Stretch Mode RHCS cluster during site down -> recovery

CEPH-83571705 - Network failures and Delays during rebalance on RHCS cluster

Polarion test : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83571705

### CODE REVIEW

**Comment by pdhiran** [comment]
Pass logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-6NTCRT

File: ceph/rados/core_workflows.py [hunk 1]
```diff
@@ -2304,33 +2304,108 @@ def warning_check(timeout):
         log.info("Daemon crash warning found in ceph health")
         return True
 
-    def configure_host_as_client(self, host_node):
+    def add_network_delay_on_host(
```
Comment (SrinivasaBharath) [inline ceph/rados/core_workflows.py:2307]: The logic looks good to me.

**Review by SrinivasaBharath — DISMISSED** [review]

**Comment by neha-gangadhar** [comment]
Looks good. Could you please run the stretch mode test as well and attach logs and we can go for merge.

**Comment by pdhiran** [comment]
> Looks good. Could you please run the stretch mode test as well and attach logs and we can go for merge.

Added logs for both stretch mode and normal RHCS cluster

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**